src/lib/functions/utils.js

```diff
@@ -10,14 +8,6 @@ const DEFAULT_LAMBDA_OPTIONS = {
 
 const SECONDS_TO_MILLISECONDS = 1000
 
-const detectAwsSdkError = ({ error, warn }) => {
-  const isAwsSdkError = error.errorMessage && error.errorMessage.includes("Cannot find module 'aws-sdk'")
-
-  if (isAwsSdkError) {
-    warn(getLogMessage('functions.missingAwsSdk'))
-  }
-}
-
```

Comment:
Would it be possible to split this into a separate PR?
I would like to get some input from the author of those lines to make sure this was not unintentionally left as dead code. Thanks!

Reply:
Removed it from here. Will create a separate PR.

Reply:
Thanks!